src/brain-even.js

```diff
@@ -0,0 +1,39 @@
+import readlineSync from 'readline-sync';
+
+console.log('Welcome to the Brain Games!');
+const userName = readlineSync.question('May I have your name? ');
+console.log(`Hello, ${userName}!`);
+
+const brainEven = () => {
+  console.log('Answer "yes" if the number is even, otherwise answer "no".');
+  const numberOfRound = 3;
```

Comment:
This should be outside the function so that the variable is not created every time the function is run.

Reply:
thanks, fixing